templates/SilverStripe/Discoverer/Service/Results/Suggestions.ss

```diff
@@ -0,0 +1,13 @@
+<ul class="discoverer-suggestions__list">
+    <% loop $Me %>
+        <% if $Up.DesiredUrl && $Up.DesiredQueryField %>
+            <li class="discoverer-suggestion">
+                <a href="{$Up.DesiredUrl}?{$Up.DesiredQueryField}={$Me}"
```

Comment:
I'm not getting links on this atm but it seems it should have been set by the conroller? 
![image](https://github.com/user-attachments/assets/b4ebef68-7ed2-43a9-b912-7f13880cb0de)